src/test/java/hw2/base/TestBase.java

```diff
@@ -0,0 +1,60 @@
+package hw2.base;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+public class TestBase {
+
+    protected WebDriver driver;
+
+    protected final static String URL = "https://jdi-testing.github.io/jdi-light/index.html";
+
+    protected WebDriverWait wait;
+
+    @BeforeMethod
+    public void setUp() {
+        WebDriverManager.chromedriver().setup();
+        driver = new ChromeDriver();
+        wait = new WebDriverWait(driver, 5);
+    }
+
+    @AfterMethod
+    public void tearDown() {
+        if (driver != null) {
+            driver.quit();
+        }
+    }
+
+    public void openTestSite(){
+        driver.get(URL);
+        driver.manage().window().maximize();
+    }
+
+    public void login(String user, String pass){
+        wait.until(ExpectedConditions.presenceOfElementLocated(By.id("user-icon"))).click();
+        wait.until(ExpectedConditions.attributeToBe(By.className("uui-profile-menu"), "class",
+                "dropdown uui-profile-menu open"));
+        wait.until(ExpectedConditions.presenceOfElementLocated(By.id("name"))).sendKeys(user);
+        wait.until(ExpectedConditions.presenceOfElementLocated(By.id("password"))).sendKeys(pass);
+        wait.until(ExpectedConditions.presenceOfElementLocated(By.id("login-button"))).click();
```

Comment:
This is a good catch to use **Explicit waits** in the test. Please read carefully the *Selenium Java Documentation* about methods which do you try to use. 
https://www.selenium.dev/selenium/docs/api/java/
```
public static ExpectedCondition<WebElement> presenceOfElementLocated(By locator)

An expectation for checking that an element is present on the DOM of a page. This does not necessarily mean that the element is visible.

Parameters:
    locator - used to find the element
Returns:
    the WebElement once it is located 
```

Please check all places where you use it

Reply:
Got it. Added proper wairs.